recipes/fuse_mnt.rb

```diff
@@ -0,0 +1,160 @@
+include_recipe "hops::default"
+
+# Intall the needed packages
+case node['platform_family']
+when 'debian'
+  package 'fuse'
+  package 'libfuse-dev'
+when 'rhel'
+  package 'fuse'
+  package 'fuse-libs'
+end
+
+# Allow non root users to mount file systems 
```

Comment:
You need to check if /etc/fuse.conf file exists first. 
If you want to override that, add a chef attribute for that.
hops/fuse_reformat = "false" 
by default. 

# bash
if [ -e /etc/fuse.conf ] && [  "#{node['hops']['fuse_reformat']}" == "false" ] ; then
   exit 1
fi
echo "user_allow_other" >>  /etc/fuse.conf

Reply:
/etc/fuse.conf should be there. If it is not then it means that installing packages in the previous section has failed and the recipes will fail. 

fuse_reformat thingy is not correct / don't a agree